Tensile/KernelWriterAssembly.py

```diff
@@ -7282,19 +7286,28 @@ def localReadDo(self, kernel, bufferIdx, iui, epsi, tP):
     loopChar = self.indexChars[loopDim]
     # mfma: for AB tile in NT layout
     if kernel["MatrixInstruction"] and (kernel["ProblemType"]["DataType"].isHalf() or kernel["ProblemType"]["DataType"].isBFloat16()) and numReadsAlongK > 1:
```

Comment:
where do we check the packing code is inserted only in NT case

Reply:
Currently LDS is NT layout for both NT and TN problem case, so will need packing for fp16 and bf16.